http/server.go

```diff
@@ -87,9 +86,9 @@ func (h *handler) PublicRand(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	roundExpectedTime := beacon.TimeOfRound(h.groupInfo.Period, h.groupInfo.GenesisTime, roundN)
+	roundExpectedTime := time.Unix(h.groupInfo.GenesisTime, 0).Add(h.groupInfo.Period * time.Duration(roundN))
```

Comment:
Why not keeping the beacon method ? If possible I would prefer to use a single point to compute the time - round mappings -
For example, here round 1 is set after one period after the genesis no ? While it is not the case, the first round happens at genesis time (round 0 is the genesis block)

Reply:
was avoiding circular dependencies:
beacon depends on net already, so this keeps a lot of code from being able to use this handler if it depends on that.

Reply:
I'm not sure I see the circular dependency here - Ive changed that line locally to using the  beacon package and it works. 
Also, my first comment still holds out, I don't believe you return the correct timestamp - hence I would insist to use the same method :D 

I am totally fine to move these functions (`TimeOfRound` `NextRound` etc ) into a separate package if that is a trouble !